src/collidable.ts

```diff
@@ -0,0 +1,43 @@
+import { PhysObject } from './physObject';
+import { CanvasValues } from './constants';
+
+export class Collidable {
+  static willCollide(
+    object: PhysObject,
+    objects: Array<PhysObject>
+  ): Array<PhysObject> {
+    const results: Array<PhysObject> = [];
+    for (let i = 0; i < objects.length; i++) {
+      if (
```

Comment:
Seems like you could make use of the border function here. Something like:

```
const {top, right, bottom, left} = border(object);
if (top || right || bottom || left) results.push(object);
```

Reply:
I don't see how I could. The border function checks object edges collision with canvas edges. I could potentially combine the two functions but I don't see the benefit.

Reply:
I misunderstood what this function does :) Disregard.